test/sql-tap/lua/sqltester.lua

```diff
@@ -426,7 +426,8 @@ box.cfg{
 
 -- A lot of tests from sql-tap are very long and not designed to be limited
 require('fiber').set_max_slice(100500)
-local engine = test_run and test_run:get_cfg('engine') or 'memtx'
+local engine = test_run and test_run:get_cfg('engine') or
+               os.getenv('TEST_ENGINE')
```

Comment:
I suppose this should be `os.getenv('TEST_ENGINE') or 'memtx'` to remain the compatibility with test_run.

Reply:
The same as with `engine-tap`:

Tarantool engine is obtained via `test-run/test_run.lua` if `test_run` variable is not `nil` and when `test_run` is `nil` we use `TEST_ENGINE`. So I don't see anything bad for `test-run`. 

Also, testsuite `sql-tap` is parameterized by both engines for all tests, see:

```
[0] ~/sources/MRG/tarantool/build $ cat ../test/sql-tap/engine.cfg 
{
    "*": {
        "memtx": {"engine": "memtx"},
        "vinyl": {"engine": "vinyl"}
    }
}
```